.github/workflows/snyk-issue.yml

```diff
@@ -0,0 +1,28 @@
+name: Snyk Issue
+
+on:
+  schedule:
+    - cron: '* */12 * * *'
+
+concurrency: snyk-issue
+
+jobs:
+  snyk:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Action
+      uses: actions/checkout@v3
+      with:
+        repository: snowflakedb/whitesource-actions
+        token: ${{ secrets.whitesource_action_token }}
+        path: whitesource-actions
+    - name: Set Env
+      run: echo "repo=$(basename $GITHUB_REPOSITORY)" >> $GITHUB_ENV
+    - name: Jira Creation
+      uses: ./whitesource-actions/snyk-issue
+      with:
+        snyk_org: ${{ secrets.snyk_org_id_public_repo }}
+        snyk_token:  ${{ secrets.snyk_github_integration_token_public_repo }}
```

Comment:
we have errors in the connector repo regarding to these two parameters:
https://github.com/snowflakedb/snowflake-connector-python/actions/runs/4303850518

I check the code in the repo `whitesource-actions`, looks like the script doesn't appreciate the two inputs.